assets/out

```diff
@@ -167,7 +188,7 @@ if [[ -n "${decorate_pr}" ]]; then
 		branch_name=$(git config --get pullrequest.branch)
 	fi
 
-	scanner_opts+=" -Dsonar.pullrequest.key=${pr_id} -Dsonar.pullrequest.branch=${branch_name}"
+	scanner_opts+=" -Dsonar.pullrequest.key=${pr_id} -Dsonar.pullrequest.branch=${branch_name} -Dsonar.pullrequest.base=\"${sonar_branch_name}"
```

Comment:
sonar.pullrequest.base also needs to be passed, per https://docs.sonarqube.org/7.4/analysis/pull-request/